src/client/app/component/CommonHeader/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './style.scss';
+
+class App extends React.Component {
+  render() {
+    return (<div className = "CommonHeader">
+      <div className="Menu"></div>
+      <div className="Avatar"></div>
+    </div>);
+  }
+}
+
```

Comment:
🍬 i think it would look much better like this:
```js
class App extends React.Component {
  render() {
    return (
      <div className="CommonHeader">
        <div className="Menu"></div>
        <div className="Avatar"></div>
      </div>
    );
  }
} 
```